src/classes/STG_PanelCustomizableRollup_CTRL.cls

```diff
@@ -105,7 +105,6 @@ public with sharing class STG_PanelCustomizableRollup_CTRL extends STG_Panel {
                 // there are existing rollups. just save the setting and reschedule jobs, don't deploy.
                 STG_Panel.stgService.stgCRLP.Customizable_Rollups_Enabled__c = true;
                 stgService.saveAll();
-                UTIL_MasterSchedulableHelper.setScheduledJobs();
```

Comment:
Wondering why batch jobs scheduling is removed from NPSP Settings Customizable Rollup enablement page?

Reply:
Oops. Thank you for catching this!